src/merge.js

```diff
@@ -0,0 +1,3 @@
+import merge from 'lodash.merge';
+
+export default merge;
```

Comment:
We did agree on this on an diffed out discussion. lodash.assign seems like a better choice.